runtime/doc/lsp.txt

```diff
@@ -1589,12 +1589,16 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
 Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
 
 enable({enable}, {filter})                       *vim.lsp.inlay_hint.enable()*
-    Enables or disables inlay hints for a buffer.
+    Enables or disables inlay hints for the {filter}ed scope.
 
-    To "toggle", pass the inverse of `is_enabled()`: >lua
+    To "toggle" globally, pass the inverse of `is_enabled()`: >lua
         vim.lsp.inlay_hint.enable(not vim.lsp.inlay_hint.is_enabled())
 <
 
+    Or you can disable it individually for the current buffer: >lua
+        vim.lsp.inlay_hint.enable(false, { bufnr = 0 })
```

Comment:
this is over-explaining. examples show mechanics of usage. they generally don't need to demonstrate usage of each parameter, unless the parameter is unusual.